frontend/src/components/print/config/ActivityListConfig.vue

```diff
@@ -34,21 +35,26 @@ export default {
       }))
     },
   },
-  defaultOptions() {
+  defaultOptions(camp) {
     return {
-      periods: [],
+      periods:
+        camp.periods().items.length === 1 ? [camp.periods().items[0]._meta.self] : [],
     }
   },
   design: {
     multiple: true,
   },
   repairConfig(config, camp) {
     if (!config.options) config.options = {}
-    if (!config.options.periods) config.options.periods = []
-    const knownPeriods = camp.periods().items.map((p) => p._meta.self)
-    config.options.periods = config.options.periods.filter((period) => {
-      return knownPeriods.includes(period)
-    })
+    if (camp.periods().items.length === 1) {
```

Comment:
This code block is the same across all 5 components, I wonder if it makes sense to extract it in a mixin or common js file. Given that this was already the case before, I can also live with merging it as is.